data/golden/rr_lyr.csv

```diff
@@ -0,0 +1,101 @@
+ra,dec
```

Comment:
Should some version info be required here? Or do we expect our golden set to be immutable?

Reply:
No, I think we should be using hashes of the files themselves for versioning. As in `git hash-object data/golden/rr_lyr.csv`. This stuff will be part of the next PR.

Reply:
Heard this idea from @dannygoldstein / W&B, find it really nice!